src/Fixer/Basic/PsrAutoloadingFixer.php

```diff
@@ -135,7 +135,7 @@ protected function createConfigurationDefinition()
     /**
      * {@inheritdoc}
      */
-    protected function applyFix(\SplFileInfo $file, Tokens $tokens)
+    protected function applyFix(\SplFileInfo $file, Tokens $tokens): void
```

Comment:
i think it will break on 2.x line
let's revert that and we should auto-introduce it after dropping 5.x